diag_manager/include/fms_diag_fieldbuff_update.inc

```diff
@@ -0,0 +1,31 @@
+#undef FMS_DIAG_FBU_DATA_TYPE_
```

Comment:
We also put in the GNU Lesser General Public License stuff in the *inc and *fh files